src/hb-ot-layout.h

```diff
@@ -402,6 +402,38 @@ hb_ot_layout_feature_get_characters (hb_face_t      *face,
 				     unsigned int   *char_count    /* IN/OUT.  May be NULL */,
 				     hb_codepoint_t *characters    /* OUT.     May be NULL */);
 
+/*
+ * BASE
+ */
+
+/**
+ * hb_ot_layout_baseline_tag_t:
+ *
+ * https://docs.microsoft.com/en-us/typography/opentype/spec/baselinetags
+ *
+ * Since: REPLACEME
+ */
+typedef enum {
+  HB_OT_LAYOUT_BASELINE_TIBETAN_HANGING					= HB_TAG ('h','a','n','g'),
+  HB_OT_LAYOUT_BASELINE_IDEOGRAPHIC_CHARACTER_FACE_BOTTOM_OR_LEFT_EDGE	= HB_TAG ('i','c','f','b'),
+  HB_OT_LAYOUT_BASELINE_IDEOGRAPHIC_CHARACTER_FACE_TOP_OR_RIGHT_EDGE	= HB_TAG ('i','c','f','t'),
+  HB_OT_LAYOUT_BASELINE_IDEOGRAPHIC_EMBOX_BOTTOM_OR_LEFT_EDGE		= HB_TAG ('i','d','e','o'),
+  HB_OT_LAYOUT_BASELINE_IDEOGRAPHIC_EMBOX_TOP_OR_RIGHT_EDGE		= HB_TAG ('i','d','t','p'),
+  HB_OT_LAYOUT_BASELINE_MATHEMATICAL_CHARACTERS_CENTER			= HB_TAG ('m','a','t','h'),
+  HB_OT_LAYOUT_BASELINE_ROMAN						= HB_TAG ('r','o','m','n'),
```

Comment:
need `_TAG` in value names.  They must match enum name.

Reply:
So hb-ot-meta and hb-ot-metrics enums also?

Reply:
Yes.  Enums always have to be consistent like that.